types/react-dragula/react-dragula-tests.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import reactDragula from 'react-dragula';
```

Comment:
```suggestion
import React = require('react');
import reactDragula = require('react-dragula');
```
see my comment on `esModuleInterop`